packages/react-components/react-tag-picker/library/src/components/TagPickerControl/useTagPickerControl.tsx

```diff
@@ -122,4 +127,74 @@ export const useTagPickerControl_unstable = (
     disabled,
     invalid,
   };
+
```

Comment:
What is the connection between this `expandIcon` modification and the addition of `disabled`?! If there's none, maybe let's split that into 2 PR's also?!

Reply:
i added this to the react-tags PR since they're related!